examples/hosts/app-integration/external-data/src/model-interface/index.ts

```diff
@@ -37,17 +37,21 @@ export interface ITaskEvents extends IEvent {
 	 */
 	(event: "nameChanged", listener: () => void);
 	/**
-	 * Emitted when the priority has changed to either the incoming value or to the NONE default value.
+	 * Emitted when the priority has changed to either the source value or to the NONE default value.
```

Comment:
I'm a bit confused by these priority change statements. What is meant by the "source" value here? I'm also not sure I understand what the "NONE" default value is, or why it's special in terms of notifications.

Reply:
Oh good point. The allusion to NONE is because of a pattern I was using when I first implemented this. It needs to go away!